_ml-commons-plugin/cluster-settings.md

```diff
@@ -290,7 +290,22 @@ plugins.ml_commons.enable_inhouse_python_model: false
 ### Values
 
 - Default value: false
-- Value range: [false, true]
+- Valid values: `false`, `true`
+
+## Enable access control for connectors
+
+When set to `true`, requires an access key when making a connection to SageMaker or Hugging Face.
```

Comment:
Access control for connectors is similar with model access control, it's not related to access key because all external API needs access key, e.g. OpenAI, Cohere or SageMaker. The connector access control is more like controlling if an OpenSearch user has permission to view, update, delete a connector, it's implemented with the OpenSearch backend roles and only the users have the corresponding backend role can access to a connector.